### ISSUE TITLE
flake: "Podman run" times out

### ISSUE THREAD

**Issue by wking** [projectatomic/libpod#1040]
In a recent Travis build, we [have][1]:

```
• Failure [122.146 seconds]
Podman run
/go/src/github.com/projectatomic/libpod/test/e2e/run_test.go:16
  podman run with built-in volume image [It]
  /go/src/github.com/projectatomic/libpod/test/e2e/run_test.go:480
  Timed out after 90.000s.
  Expected process to exit.  It did not.

  /go/src/github.com/projectatomic/libpod/test/e2e/libpod_suite_test.go:334

  Full Stack Trace
  ...
 	/go/src/github.com/projectatomic/libpod/test/e2e/run_test.go:490 +0x324
  ...
```

The failing line is [in the `WaitWithDefaultTimeout` helper][2], and the time out is from [this][3]:

```go
session = podmanTest.Podman([]string{"run", "--rm", "-e", "MYSQL_USER=zmuser", "-e", "MYSQL_PASSWORD=zmpass", "-e", "MYSQL_DATABASE=zm", "-e", "MYSQL_ROOT_PASSWORD=mysqlpassword", "docker.io/centos/mariadb-101-centos7", "ls", "-al", "/var/lib/mysql/data"})
```

The tests are from #1026.

[1]: https://travis-ci.org/projectatomic/libpod/jobs/399193575#L3796
[2]: https://github.com/projectatomic/libpod/blob/d357703e061258a57854df1880e23775dbe717ef/test/e2e/libpod_suite_test.go#L334
[3]: https://github.com/projectatomic/libpod/blob/d357703e061258a57854df1880e23775dbe717ef/test/e2e/run_test.go#L489

**Comment by mheon** [comment]
This is a 154mb (compressed) image we have to pull, so I suspect that has a large part to do with the timeout.

I think skipping this test on Travis, with its known-slow network, is probably our best option.

**Comment by wking** [comment]
> This is a 154mb (compressed) image we have to pull, so I suspect that has a large part to do with the timeout.

Possible alternatives would be:

a. Bumping the timeout for this operation.
b. Running the `pull` as a separate step (possibly still with an increased timeout).
c. Reworking the test to use a smaller image (what is the test trying to demonstrate anyway?).

**Comment by mheon** [comment]
The test is attempting to demonstrate that image volumes work when the container is not run as root. The reproducer is somewhat unnecessarily long (the environment settings in particular are unnecessary) but I don't think we can test this with our standard test images. We could potentially build an image suitable for this test on top of the alpine image.

**Comment by wking** [comment]
> We could potentially build an image suitable for this test on top of the alpine image.

I'm in favor of building our own small test image.  But is there a need to involve alpine?  If this is about exercising a property of the image format (vs. a property of the root filesystem), couldn't we use an image that just contained a BusyBox binary?

**Comment by rhatdan** [comment]
Yes we should build our own image.
@umohnani8 PTAL

**Comment by umohnani8** [comment]
Fixed in #1044
